subprojects/core/src/main/java/org/gradle/internal/ToolingModelContext.java

```diff
@@ -14,23 +14,28 @@
  * limitations under the License.
  */
 
-import io.cucumber.java.en.Given;
-import io.cucumber.java.en.Then;
-import io.cucumber.java.en.When;
+package org.gradle.internal;
 
-public class HelloStepdefs {
-    @Given("^I have a hello app with Howdy and /four")
-    public void I_have_a_hello_app_with() {
-        System.out.println("Given");
+public class ToolingModelContext {
```

Comment:
💭  Would it be fair to say this is about the model _request_? Maybe we can add this bit to the class name as well.